scripts/west_commands/runners/openocd.py

```diff
@@ -47,8 +47,9 @@ def __init__(self, cfg, pre_init=None, reset_halt_cmd=DEFAULT_OPENOCD_RESET_HALT
                     search_args.append('-s')
                     search_args.append(path.dirname(i))
 
-        for p in cfg.openocd_search:
-            search_args.extend(['-s', p])
+        if cfg.openocd_search is not None:
+            for p in cfg.openocd_search:
+                search_args.extend(['-s', p])
```

Comment:
Or just this without the previous None check:
```suggestion
            for p in cfg.openocd_search or []:
                search_args.extend(['-s', p])
```